homeassistant/components/modern_forms/services.yaml

```diff
@@ -1,3 +1,28 @@
+set_light_sleep_timer:
+  name: Set light sleep timer
+  description: Set a sleep timer on a Modern Forms light.
+  target:
+    entity:
+      integration: modern_forms
+      domain: light
+  fields:
+    sleep_time:
+      name: Sleep Time
+      description: Number of seconds to set the timer.
```

Comment:
It seems to be minutes in the code and below in the unit.

Reply:
cleanup in #52058